conda-envs/windows-environment-dev-py38.yml

```diff
@@ -10,10 +10,8 @@ dependencies:
 - cloudpickle
 - fastprogress>=0.2.0
 - h5py>=2.7
+- libblas=*=*mkl
```

Comment:
What does this Syntax mean?

And why is the line only for Windows Python 3.8?
(Maybe add a comment inline)

Reply:
Oh wait, is there just one yml for Windows? (on my phone, sorry)

Reply:
That means that you want to use the mkl implementation of libblas, rather than e.g. openblas.

Reply:
I couldn't find any explanation of the `=*=*` operator, but it looks like this is the relevant docs section: https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation

(Just leaving this here for reference.)